_new_content_templates/blank_releasenotes.md

```diff
@@ -120,7 +120,7 @@ For information on using premium skins and icon packs, see: [Premium Skins and I
 
 ## Accompanying Premium self-hosted server-side component changes
 
-The {{site.productname}} vnumtxt release includes accompanying changes affecting the {{site.productname}} **self-hosted** services for the following plugins:
+The {{site.productname}} vnumtxt release includes changes affecting the {{site.productname}} **self-hosted** services for the following plugins:
```

Comment:
`vnumtxt`?

Reply:
Oh wait, this is a template. Can probably ignore?

Reply:
@metricjs that's the template file, so that's expected.